Features2/tests/features/addcontact.feature

```diff
@@ -0,0 +1,34 @@
+Feature:AddContact 
+    As a user
+    I want to add my contacts to the ContactFox
+    So can easily remember the contacts that are necessary for message
+
+    Background: User is looged In
+        Given the  user  navigated to the login page
+        When the user submit username and password
+        Then the user should be logged in to the homepage
+
+        Scenario:add to contacts with personal contact type
+        Given the user is navigated to the home page
+        When the user submit name with "name" and email with "valid email" and chooses the contact type as "personal"
+        Then the user info will be added to the contact list  with the personal  badge
```

Comment:
which user to check ? may be give user information that has been added to contact